crates/tuono/tests/utils.rs

```diff
@@ -18,6 +18,22 @@ impl TempTuonoProject {
 
         env::set_current_dir(temp_dir.path()).expect("Failed to change current dir into temp_dir");
 
+        let project = TempTuonoProject {
+            original_dir,
+            temp_dir,
+        };
+
+        project.add_file("./tuono.config.ts");
+
+        project
+    }
+
+    pub fn new_with_no_config() -> Self {
```

Comment:
I think duplicating the function is not a good pattern. I'd consider reusing it within the `new` method like:

```rs
pub fn new() -> Self {
        let project = TempTuonoProject::new_with_no_config();

        project.add_file("./tuono.config.ts");

        project
    }
```